CHANGELOG.md

```diff
@@ -21,6 +21,9 @@
 * Display the fiscal year in which an order was opened. Fixes UIOR-1403.
 * Automatically set to `Independent order and receipt quantity` if a user selects `Receipt not required` for opened order. UIOR-1243.
 * *BREAKING* Update for Split Search & Browse APIs. Refs UIOR-1452.
+* Add deprecated checkbox for order prefix and suffix and make it editable. UIOR-1450.
+* Only show not deprecated prefixes and suffixes on order creation. UIOR-1450.
+* On order edit, show a hint if the prefix or suffix is deprecated. UIOR-1450.
```

Comment:
Could you please condense this into a single entry? In the changelog, we usually keep one entry per ticket. If the work is unrelated, it might be better not to include it under the same ticket.